bee-message/src/payload/fpc/conflict.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::payload::transaction::TransactionId;
+
+use bee_packable::Packable;
+
+/// Describes a vote in a given round for a transaction conflict.
+#[derive(Clone, Debug, Eq, PartialEq, Packable)]
+#[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
+pub struct Conflict {
+    /// ID of the conflicting transaction.
+    transaction_id: TransactionId,
+    /// The nodes opinion value in a given round.
```

Comment:
Just checking, is this supposed to be the opinion of a specific node or several nodes ?
`nodes` vs `node's` ?

Reply:
It should be `node's`.